sql/views.sql

```diff
@@ -259,5 +259,124 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
     LEFT OUTER JOIN LATERAL @extschema@.data_node_hypertable_info(CASE WHEN s.node_up THEN s.node_name ELSE NULL END) size ON TRUE
   GROUP BY s.node_name, s.node_up, s.owner, s.options;
 
+-- chunks metadata view, shows information about the primary dimension column
+CREATE OR REPLACE VIEW  timescaledb_information.chunks
+AS
+SELECT 
+       hypertable_schema, hypertable_name,
```

Comment:
I see that the indentation is different from the rest of the file or PG. Can it be good to follow some standard? I know that @svenklemm uses pgFormatter. Should we use it?

Reply:
I would like to run formatting tools at the end of this series of PRs. I need a final clean-up PR to remove old functionality.

Reply:
used a formatting tool for that query. Plan to do that for entire file after PR clean-up.